File: EjerciciosEPI/4_lista_pares/lista_pares.c
#include <stdio.h>

int main () {
	int a;
	int b;

	printf("Introduzca el primer numero: ");
	scanf("%d", &a);

	printf("Introduzca el segundo numero: ");
	scanf("%d", &b);

	printf("Numeros pares entre %d y %d:\n", a, b);

	for (int i = a ; i <= b ; i++)
	{
		if (i % 2 == 0)
		{
			printf("%d\n", i);
		}
	}
	return 0;
}

File: EjerciciosEPI/4_lista_pares/lista_pares.py
"""
Programa que muestre la lista de pares entre dos numeros (ambos incluidos) pedidos por teclado.
Ejemplo:
    Introduzca el primer numero: 10
    Introduzca el segundos numero: 15
    Numeros pares entre 10 y 15:
    10
    12
    14
"""

a = 0
b = 0

a = int(input("Introduzca el primer numero: "))
b = int(input("Introduzca el segundo numero: "))

print ("Numeros pares entre", a, "y", b, ":")
for i in range (a, b+1, 1):
    if (i % 2 == 0):
        print(i)
